.rubocop-https---raw-githubusercontent-com-everypolitician-everypolitician-data-master--rubocop-base-yml

```diff
@@ -0,0 +1,59 @@
+# A base Rubocop configuration that other repos can inherit from
```

Comment:
Why is this file here? This seems like you've just done a bulk add and commit without checking what's going in…